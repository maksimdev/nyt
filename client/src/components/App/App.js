import React, { Component } from 'react';
import './App.css';
import SearchContainer from '../SearchContainer';

class App extends Component {
  render() {
    return (
      <div className="app">
        <main className="main">
          <h1>The New York Times Article Search!</h1>
          <SearchContainer />
        </main>
      </div>
    );
  }
}

export default App;
