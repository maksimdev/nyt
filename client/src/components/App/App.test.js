import React from 'react';
import * as Enzyme from 'enzyme';
import App from './App';
import Adapter from 'enzyme-adapter-react-16';
import enzymeSerializer from 'enzyme-to-json/serializer';

Enzyme.configure({ adapter: new Adapter() })

expect.addSnapshotSerializer(enzymeSerializer);
describe('App', () => {
    it('should to match snapshot', () => {
        const AppComponent = Enzyme.shallow(<App />);
        expect(AppComponent).toMatchSnapshot();
    });
});