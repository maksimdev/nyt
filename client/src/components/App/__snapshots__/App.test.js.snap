// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should to match snapshot 1`] = `
<div
  className="app"
>
  <main
    className="main"
  >
    <h1>
      The New York Times Article Search
    </h1>
    <Connect(SearchContainer) />
  </main>
</div>
`;
